packages/forma-36-react-components/src/components/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -174,6 +177,8 @@ export class Dropdown extends Component<DropdownProps, DropdownState> {
           <DropdownContainer
             anchorDimensionsAndPositon={this.state.anchorDimensionsAndPositon}
             position={this.props.position}
+            className={`${className}__container`}
```

Comment:
that's a bit _unconventional_ I think, `className` might be smth like `f36-margin-top--m f36-margin-bottom--m`, right? do we really need it after exposing `ref`

Reply:
Totally didn't think about that case - good point.